### ISSUE TITLE
TypeError: Cannot set property 'publicPath' of undefined

### ISSUE THREAD

**Issue by limonte** [webpack-contrib/karma-webpack#420]
```
  TypeError: Cannot set property 'publicPath' of undefined
    at Plugin (/node_modules/karma-webpack/dist/karma-webpack.js:221:39)
```

https://github.com/webpack-contrib/karma-webpack/blob/master/src/karma-webpack.js#L227

Looks like `webpackMiddleware` is `undefined` by default instead of `{}`

**Comment by rensbaardman** [comment]
Yep, can confirm this. Was introduced somewhere between v4.0.0-rc.6 and v4.0.1.

**Comment by ryanclark** [comment]
This was fixed in #418, but I'm waiting for the CLA check to pass before I can merge

**Comment by ryanclark** [comment]
Fixed in v4.0.2
